test/e2e/report_dynamic_input_data_test.go

```diff
@@ -163,3 +170,145 @@ func testReportsProduceData(t *testing.T, testReportingFramework *reportingframe
 		})
 	}
 }
+
+func testReportIsDeletedWhenNoDeps(t *testing.T, testReportingFramework *reportingframework.ReportingFramework) {
+	// cron schedule to run every minute
+	cronSchedule := &metering.ReportSchedule{
+		Period: metering.ReportPeriodCron,
+		Cron: &metering.ReportScheduleCron{
+			Expression: "*/1 * * * *",
+		},
+	}
+	start := time.Now()
+	var foundOnce = false
```

Comment:
You can just specify `var foundOnce bool` here as Golang has this notion of default types for empty variables.

Reply:
ah, yeah simpler, thanks.